### ISSUE TITLE
Focus new parameters in procedure editor

### ISSUE THREAD

**Issue by nathan** [LLK/scratch-flash#414]
When a label is added in the procedure editor, it correctly receives keyboard focus so the user can begin typing the label. When a parameter is added, it should be focused too.
